class Failure {
  final String message;

  Failure([this.message = 'An unexpected method has occurred']);
}
